openslides_backend/shared/interfaces/event.py

```diff
@@ -9,6 +9,9 @@ class EventType(str, Enum):
     Update = "update"
     Delete = "delete"
 
+    def __repr__(self) -> str:
+        return repr(self.value)
```

Comment:
For easier copying of the printed write requests - should not interfere with anything